### PR TITLE
Feature/cg 486

### PR DESCRIPTION
**Pull request by tyler-dunkel**

### CODE REVIEW

File: src/services/service/format.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      loadBalancerClass,
+      loadBalancerIP: loadBalancerIp,
+      loadBalancerSourceRanges,
+      ports,
+      publishNotReadyAddresses,
+      selector,
+      sessionAffinity,
+      sessionAffinityConfig,
+      type,
+    },
```
Comment (mfranceschit) [inline src/services/service/format.ts]: We could use some safe accessor here.

File: src/services/service/format.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    status: { conditions, loadBalancer },
```
Comment (mfranceschit) [inline src/services/service/format.ts]: same here.

Comment (tyler-dunkel) [reply src/services/service/format.ts]: done

File: src/services/service/format.ts (hunk 1 above)
Comment (tyler-dunkel) [reply src/services/service/format.ts]: done

**Review by mfranceschit — APPROVED** [review]

**Pull request merged**